### PR TITLE
NZ data - fix parser

### PR DESCRIPTION
**Pull request by drc38**
The changes may need refining as I am not sure how the 'NZ' zone key will be handled with the existing keys on the map, see #3562 for discussion on the NZ parser being offline

### CODE REVIEW

**Comment by drc38** [comment]
@skovhus, are you able to review this PR please?

**Comment by Eonsim** [comment]
Any update on this one? Would be nice to get it up and running again for NZ.

**Comment by leon-v** [comment]
Anything we can do to speed up the process / help ? :-)

**Review by madsnedergaard — COMMENTED** [review]
Sorry for the silence, we're finally ready internally to get this merged! 🥳 
One comment on the parser and a merge of `master` - and then we're good to go!

File: parsers/NZ.py [hunk 1]
```diff
@@ -29,49 +29,50 @@ def fetch(session=None):
...
-        'source': 'em6live.co.nz',
+        'source': 'api.em6.co.nz',
         'zoneKey': zone_key
     }
 
-def fetch_production(zone_key=None, session=None, target_datetime=None, logger=None) -> dict:
+def fetch_production(zone_key='NZ', session=None, target_datetime=None, logger=None) -> dict:
     """Requests the last known production mix (in MW) of a given zone."""
     if target_datetime:
         raise NotImplementedError('This parser is not able to retrieve data for past dates')
```
[the remark this answers is not in the document]
Comment (drc38) [reply parsers/NZ.py:78]: That's odd as it was part of the original code, change made 😉

**Comment by drc38** [comment]
@madsnedergaard I assume this is unrelated to my changes? 
`FAIL: test_process_solar_rooftop (test_OPENNEM.TestOPENNEM)`

**Comment by madsnedergaard** [comment]
> @madsnedergaard I assume this is unrelated to my changes? 
> 
> `FAIL: test_process_solar_rooftop (test_OPENNEM.TestOPENNEM)`

Yeah, it's another parser - I'll restart the CI job which should fix it :)

**Comment by drc38** [comment]
@madsnedergaard anything else needed or is it good to merge?

**Review by madsnedergaard — APPROVED** [review]

**Pull request merged**